recipes/r-easy.utils/recipe.yaml

```
@@ -0,0 +1,52 @@
+schema_version: 1
+
+context:
+  name: easy.utils
+  version: 0.1.0
+
+package:
+  name: r-${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://cran.r-project.org/src/contrib/easy.utils_${{ version }}.tar.gz
+  sha256: 86e3ece6f692868113ca444c3261c73c6d1be5db7dfea6634bf32ca9ce85e606
+
+build:
+  number: 0
+  script: R CMD INSTALL --build .
+  merge_build_and_host_envs: true
+  dynamic_linking:
+    rpaths:
+      - lib/R/lib/
+      - lib/
+
+requirements:
+  host:
+    - r-base
+    - r-dplyr
+    - r-rlang
+    - r-fastmatch
+    - r-scales
+  run:
+    - r-base
+    - r-dplyr
+    - r-rlang
+    - r-fastmatch
+    - r-scales
+
+tests:
+  - script:
+      - if: win
+        then: Rscript -e \"library('easy.utils')\"
+        else: Rscript -e  \"library\('easy.utils'\)\"
+
+about:
+  summary: "Some utility functions for validation and data manipulation. These functions can be helpful to reduce internal codes everywhere in package development."
+  license: MIT
+  license_file: LICENSE
```

Original license is `License: MIT + file LICENSE`, so the MIT license from `r-base` should be bundled:

```suggestion
  license_file:
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/MIT
    - LICENSE
```

Maybe this can be instead `${{ PREFIX }}/lib/R/share/licenses/MIT`?